Sources/PixelExperimentKit/PixelExperimentKit.swift

```diff
@@ -82,7 +82,7 @@ extension PixelKit {
     /// This function:
     /// 1. Validates if the experiment is active.
     /// 2. Ensures the user is within the specified conversion window.
-    /// 3. Tracks actions performed and sends the pixel once the target value is reached (if applicable).
+    /// 3. Sends the pixel if not sent before (unique by name and parameter)
```

Comment:
Should we not be storing the pixel that was sent in this method so that we can ensure pixels are unique? (i.e not sent again if they were sent before). When the value being sent is unchanged, this method fires the same pixel on subsequent calls.